### PR TITLE
Add quickstart with kind and nginx

### PR DESCRIPTION
**Pull request by oscr**
Hi :wave: 

Great project! Here is a draft for a little larger quickstart example to just show of how awesome this project is. 

Any and all feedback is welcome

Best

### CODE REVIEW

File: docs/quick-start.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+kube-system                            Active   7m5s
+local-path-storage                     Active   7m2s
+vcluster                               Active   48s
+```
+
+`vcluster` is the namespace for our virtual cluster. Now lets attempt to use it.
+
+**Note! Keep the following command running for the duration of this tutorial**
+```shell
+vcluster connect kind -n vcluster
```
Comment (matskiv) [inline docs/quick-start.md:97]: A tip: if you were to create the vcluster with the NodePort service, the connect command would set up kube config without port-forwarding and exit. This only works on some local clusters, but it should work with kind. The required helm values:
```
service:
  type: NodePort
```

**Review by matskiv — APPROVED** [review]
Awesome! Thank you for your contribution.
I am happy to merge it as is, but you called this a draft, so just let me know if you plan to add anything else to this, or whether I can go ahead and merge. :)

File: docs/quick-start.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+```shell
+clusterctl init --infrastructure vcluster
+```
+
+Next we will create our virtual cluster within our `kind` cluster.
+```shell
+export CLUSTER_NAME=kind
+export CLUSTER_NAMESPACE=vcluster
+# Note same version as kind cluster above
+export KUBERNETES_VERSION=1.23.4
```
[the remark this answers is not in the document]
Comment (oscr) [reply docs/quick-start.md:66]: Thank you! Then I will remove that comment since it's misleading.

**Comment by oscr** [comment]
> Awesome! Thank you for your contribution. I am happy to merge it as is, but you called this a draft, so just let me know if you plan to add anything else to this, or whether I can go ahead and merge. :)

Great to hear! I will try out your advice, read everything again and push the changes. 

Should I squash my commits as is the case in CAPI project?

**Comment by matskiv** [comment]
Sounds good, thank you.
Yes, squashing commits would be nice :)

**Comment by matskiv** [comment]
Oh, sorry @oscr, we merged the PR too soon.
Feel free to submit a follow-up PR if decide to make those changes.
Sorry for the inconvenience.

**Comment by oscr** [comment]
No problem at all @matskiv! I will just make a new pr with the suggested changes. Thanks for the quick feedback!

**Pull request merged**